src/CategoryList.js

```diff
@@ -1,34 +1,46 @@
-import {useEffect, useState, useContext} from "react";
-import { useBetween } from 'use-between';
-import Select from 'react-select';
-import useShareableState from "./Utils";
-import {MainContext} from "./Utils";
-// import NoteList from "./NoteList";
-
-const CategoryList = (props) => {
-    const {categories, setCategories, setCategoryId} = useBetween(useShareableState);
-    useEffect(() => {
-        fetch('https://thingproxy.freeboard.io/fetch/https://pti.anims.club/getCategories')
-            .then(response => response.json())
-            .then(data => {
-                setCategories(data);
-            });
-    }, []);
-    const options = [];
-    categories.forEach(category => {
-        options.push({id: category.id, value: category.name, label: category.name});
-    });
-    const changeHandler = (param) => {
-        // console.log("Handler triggered!");
-        setCategoryId(param.id);
-    };
-    const value = useContext(MainContext);
-    if(value!==undefined){
-        setCategoryId(value);
-    }
-    return (
-        <Select options={options} onChange={(value) => {changeHandler(value)}}/>
-    );
-}
-
-export default CategoryList;
\ No newline at end of file
+import { useEffect, useState, useContext } from "react";
+import { useBetween } from "use-between";
+import Select from "react-select";
+import useShareableState from "./Utils";
+import { MainContext } from "./Utils";
+// import NoteList from "./NoteList";
+
+const CategoryList = (props) => {
+  const { categories, setCategories, setCategoryId } =
+    useBetween(useShareableState);
+  useEffect(() => {
+    fetch(
+      "https://thingproxy.freeboard.io/fetch/https://pti.anims.club/getCategories"
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        setCategories(data);
+      });
+  }, []);
+  const options = [];
+  categories.forEach((category) => {
+    options.push({
+      id: category.id,
+      value: category.name,
+      label: category.name,
+    });
+  });
+  const changeHandler = (param) => {
+    // console.log("Handler triggered!");
+    setCategoryId(param.id);
+  };
+  const value = useContext(MainContext);
+  if (value.categoryId !== undefined) {
+    setCategoryId(value.categoryId);
+  }
+  return (
+    <Select
+      options={options}
+      onChange={(value) => {
+        changeHandler(value);
+      }}
```

Comment:
I guess there is no need for empty function here
```suggestion
      onChange={changeHandler}
```